draft-ietf-tls-dtls-connection-id.md

```diff
@@ -51,22 +51,25 @@ author:
 
 normative:
   RFC2119:
-  I-D.ietf-tls-dtls13:
-  RFC6347:
   RFC5246:
+  RFC6347:
 informative:
   RFC6973:
+  I-D.ietf-tls-dtls13:
+
 --- abstract
 
-This document specifies the "Connection ID" concept for the Datagram Transport Layer Security
-(DTLS) protocol, version 1.2 and version 1.3.
+This document specifies the Connection ID construct for the Datagram Transport
+Layer Security (DTLS) protocol version 1.2.  {{I-D.ietf-tls-dtls13}} specifies
```

Comment:
Same comment about 1.2 and below. I tend to think we should just say DTLS before 1.3.

Reply:
See reply above.

Reply:
I don't think we need to explicitly scope it to 1.2.

Reply:
OK, that makes sense.  DTLS 1.3 will then supersede this mechanism with its own CID machinery.